diff --git a/node_modules/randomfill/browser.js b/node_modules/randomfill/browser.js
index ce34a69..85ad850 100644
--- a/node_modules/randomfill/browser.js
+++ b/node_modules/randomfill/browser.js
@@ -7,7 +7,7 @@ var safeBuffer = require('safe-buffer')
 var randombytes = require('randombytes')
 var Buffer = safeBuffer.Buffer
 var kBufferMaxLength = safeBuffer.kMaxLength
-var crypto = global.crypto || global.msCrypto
+var crypto = globalThis.crypto || globalThis.msCrypto
 var kMaxUint32 = Math.pow(2, 32) - 1
 function assertOffset (offset, length) {
   if (typeof offset !== 'number' || offset !== offset) { // eslint-disable-line no-self-compare
